packages/cli/src/eventbus/message-event-bus-destination/message-event-bus-destination-webhook.ee.ts

```diff
@@ -95,7 +100,56 @@ export class MessageEventBusDestinationWebhook
 		if (options.sendPayload) this.sendPayload = options.sendPayload;
 		if (options.options) this.options = options.options;
 
+		const axiosSetting = this.buildAxiosSetting(options);
+
+		this.axiosInstance = axios.create(axiosSetting);
+
 		this.logger.debug(`MessageEventBusDestinationWebhook with id ${this.getId()} initialized`);
+		this.logger.debug(
+			`MessageEventBusDestinationWebhook with options ${JSON.stringify(this.options)}`,
+		);
+	}
+
+	private buildAxiosSetting(
+		options: MessageEventBusDestinationWebhookOptions,
+	): CreateAxiosDefaults {
+		const axiosSetting: CreateAxiosDefaults = {
+			headers: {},
+			method: options.method as Method,
+			url: options.url,
+			maxRedirects: 0,
+		} as AxiosRequestConfig;
+
+		if (options.options?.redirect?.followRedirects) {
+			axiosSetting.maxRedirects = options.options.redirect?.maxRedirects;
+		}
+
+		axiosSetting.proxy = options.options?.proxy;
+
+		axiosSetting.timeout = options.options?.timeout ?? 5 * Time.seconds.toMilliseconds;
+
+		const agentOptions: HTTPAgentOptions = {
+			// keepAlive to keep TCP connections alive for reuse
+			keepAlive: options.options?.socket?.keepAlive ?? true,
+			maxSockets: options.options?.socket?.maxSockets ?? 50,
+			maxFreeSockets: options.options?.socket?.maxFreeSockets ?? 50,
+			maxTotalSockets: options.options?.socket?.maxSockets ?? 50,
+			// Socket timeout in milliseconds defaults to 5 seconds
+			timeout: options.options?.timeout ?? 5 * Time.seconds.toMilliseconds,
```

Comment:
Pull out to some constants?

```typescript
const DEFAULT_MAX_SOCKETS = 50;
const DEFAULT_TIMEOUT_MS = 5 * Time.seconds.toMilliseconds;
```

```suggestion
maxSockets: options.options?.socket?.maxSockets ?? DEFAULT_MAX_SOCKETS,
maxFreeSockets: options.options?.socket?.maxFreeSockets ?? DEFAULT_MAX_SOCKETS,
maxTotalSockets: options.options?.socket?.maxSockets ?? DEFAULT_MAX_SOCKETS,
timeout: options.options?.timeout ?? DEFAULT_TIMEOUT_MS,
```